row_067_ID.py

```
@@ -0,0 +1,109 @@
+           filename = secure_filename(f.filename)           
+           #Save to passporteye folder to process
+           os.chdir("C:/Users/shahi/AppData/Local/Programs/Python/Python36-32/Lib/site-packages/passporteye")
+           f.save(filename)
+           mrz = read_mrz(filename)
+           mrz_data = mrz.to_dict()
+           os.remove(filename)
+           return json.dumps(mrz_data)
+
+
+if __name__ =='__main__':
+    app.run(debug=True)
```

Ruff (0.13.2)

Use of `debug=True` in Flask app detected

(S201)